packages/wallets/coin98/src/adapter.ts

```diff
@@ -158,7 +158,37 @@ export class Coin98WalletAdapter extends BaseSignerWalletAdapter {
         const signedTransactions: Transaction[] = [];
         for (const transaction of transactions) {
             signedTransactions.push(await this.signTransaction(transaction));
+            await this.sleep()
         }
         return signedTransactions;
     }
+
+
+    async signMessage(message: Uint8Array): Promise<Uint8Array> {
+        try {
+            const wallet = this._wallet;
+            if (!wallet) throw new WalletNotConnectedError();
+
+            try {
+                const decodedMessage = new TextDecoder().decode(message)
+                
+                const response = await wallet.request({ method: 'sol_sign', params: [decodedMessage] });
+
+                const signature = new TextEncoder().encode(response.sig);
```

Comment:
Same here, can we use `response.signature` and Base64?

Reply:
Oh, base58 might be better, as that's what's used for standard Solana transaction signatures.